openlibrary/plugins/openlibrary/lists.py

```diff
@@ -244,28 +235,33 @@ def get_list_data(self, lst):
             }
         }
 
-    def dumps(self, data):
-        web.header("Content-Type", self.content_type)
-        return formats.dump(data, self.encoding)
 
-class list_view_yaml(list_view_json):
-    encoding = "yml"
-    content_type = "text/yaml"
-
-class list_seeds(delegate.page):
-    path = r"(/people/\w+/lists/OL\d+L)/seeds"
+class list_view_json(delegate.page):
+    path = r"(/people/[^/]+/lists/OL\d+L)"
     encoding = "json"
-
     content_type = "application/json"
 
     def GET(self, key):
-        lst = web.ctx.site.get(key)
-        if not lst:
+        i = web.input()
+        raw = i.get("_raw") == "true"
+        lst = get_list(key, raw=raw)
+        if not lst or lst.type.key == '/type/delete':
```

Comment:
```suggestion
        if not lst or lst['type']['key'] == '/type/delete':
```
This is throwing an attribute error on testing.  Even with the above change, a KeyError is thrown when `raw` is false.  There should either be another check for `lst['links']['self'] here when `raw is false, or `type.key` should be added to the dictionary returned by `get_list()`.